assets/js/modules/analytics/datastore/index.js

```diff
@@ -16,23 +16,20 @@
  * limitations under the License.
  */
 
-/**
- * External dependencies
- */
-
 /**
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-import { createModuleStore } from 'assets/js/googlesitekit/modules/create-module-store';
+import Modules from 'googlesitekit-modules';
 import accounts from './accounts';
 import properties from './properties';
 import profiles from './profiles';
+import settings from './settings';
 import tags from './tags';
+import { STORE_NAME } from './constants';
+export { STORE_NAME };
 
-export const STORE_NAME = 'modules/analytics';
-
-const baseModuleStore = createModuleStore( 'analytics', {
+const baseModuleStore = Modules.createModuleStore( 'analytics', {
 	settingSlugs: [
```

Comment:
We should pass `storeName: STORE_NAME` here. `createModuleStore` also handles creating the module store name by default, but it supports passing one if it's preferred to define elsewhere. It doesn't matter functionality-wise whether we pass it here or not, but we should probably respect our own API. By passing it, we ensure that `baseModuleStore.STORE_NAME` also uses that value.

Reply:
✅